core/backend/src/ElementAspect.ts

```diff
@@ -147,7 +147,7 @@ export class ExternalSourceAspect extends ElementMultiAspect {
    * @note Warning: in a future major release the `scope` property will be optional, since the scope is intended to be potentially invalid.
    *       all references should treat it as potentially undefined, but we cannot change the type yet since that is a breaking change.
    */
-  public scope: RelatedElement;
+  public scope: RelatedElement | undefined;
```

Comment:
this is potentially breaking, please investigate the ramifications

Reply:
No "potential" about it - it violates our [API support policies](https://www.itwinjs.org/learning/api-support-policies/).
Consider deprecating it and adding a new property.